### PR TITLE
add files

### PR DESCRIPTION
**Pull request by ChaikaS**

### CODE REVIEW

File: React(Simple layout)/simple_layout/src/App.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <div>
+      <Header user={user} />
+      <Content avatar={avatar} />
+      <Footer />
+    </div>
+  );
+}
+
+export default App;
```
Comment (mikalaider) [inline React(Simple layout)/simple_layout/src/App.js]: Let's create an additional folder inside the Components folder for the `App` component and put all the related files

File: React(Simple layout)/simple_layout/src/Components/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    firstName: "Sergei",
+    lastName: "Chaikovsky",
+  };
+  return (
+    <div>
+      <Header />
+      <UserInfo user={user} />
+    </div>
+  );
+}
```
Comment (mikalaider) [inline React(Simple layout)/simple_layout/src/Components/HomePage.js]: The same here

File: React(Simple layout)/simple_layout/src/App.js (hunk 1 above)
Comment (ChaikaS) [reply React(Simple layout)/simple_layout/src/App.js]: done

File: React(Simple layout)/simple_layout/src/Components/HomePage.js (hunk 2 above)
Comment (ChaikaS) [reply React(Simple layout)/simple_layout/src/Components/HomePage.js]: done

**Pull request merged**